topofileformats/asd.py

```diff
@@ -170,9 +170,10 @@ def load_asd(file_path: Path, channel: str):
     version please either look into the `read_header_file_version_x` functions or print the keys too see what metadata
     is available.
     """
-    # Binary mode open does not take an encoding argument
-    # pylint: disable=unspecified-encoding
-    with Path.open(file_path, "rb") as open_file:
+    # Ensure the file path is a Path object
+    file_path = Path(file_path)
+    # Open the file in binary mode
+    with Path.open(file_path, "rb", encoding=None) as open_file:  # pylint: disable=W1514
```

Comment:
I prefer using the description as its more informative to readers and they don't have to go and look up what [W1514](https://pylint.pycqa.org/en/latest/user_guide/messages/warning/unspecified-encoding.html) is.

```suggestion
    with Path.open(file_path, "rb", encoding=None) as open_file:  # pylint: disable=unspecified-encoding
```